@navikt/ds-css/tag.css

```
@@ -1,12 +1,18 @@
 .navds-tag {
-  border-radius: 4px;
   border: var(--navds-border-default);
+  border-radius: 4px;
   background-color: var(--navds-color-tag-info-background);
   border-color: var(--navds-color-tag-info-border);
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.125rem var(--navds-spacing-3);
+  min-height: 32px;
+}
 
-  /* Padding calculated based on 16px font for 24px tall tag */
-  padding: 0.0625rem var(--navds-spacing-2);
+.navds-tag--small {
+  min-height: 24px;
+  padding: 0.125rem var(--navds-spacing-1);
```

Denne endringen gjør det tydeligere at vi arver padding top/bottom fra `.navds-tag`

```suggestion
  padding-right: var(--navds-spacing-1);
  padding-left: var(--navds-spacing-1);
```